wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/wdlom2wom/graph/CallElementToGraphNode.scala

```diff
@@ -38,10 +38,14 @@ object CallElementToGraphNode {
       * @return ErrorOr of LocalName(key) mapped to ExpressionNode(value).
       */
     def expressionNodeMappings(callable: Callable): ErrorOr[Map[LocalName, AnonymousExpressionNode]] = {
+      def validInput(name: String, definition: Callable.InputDefinition): Boolean = {
+        definition.name == name && !definition.isInstanceOf[FixedInputDefinition]
+      }
+
       a.node.body match {
         case Some(body) =>
           body.inputs.map(input => input.key -> input.value).toMap.traverse {
-            case (name, expression) =>
+            case (name, expression) if callable.inputs.exists(validInput(name, _)) =>
```

Comment:
[ToL] It wasn't immediately clear to me what value this `_` binds to - the result of the `traverse`? It has some competing `_`'s in close proximity, so it might be nice to give this one a name

Reply:
Sure! In fact it was the `.exists` rather than the `traverse` - which IMO reinforces your point that it was non-obvious!